modules/programs/ssh.nix

```diff
@@ -15,6 +15,12 @@ let
 
   unwords = builtins.concatStringsSep " ";
 
+  mkSetEnvStr = envStr: unwords
+    (mapAttrsToList
+      (name: value: "${name}=${value}")
```

Comment:
`value` looks like it needs to be escaped.

Reply:
The man page is unclear about what escaping is possible. Testing would be needed.

Reply:
I have wrapped the `value` in `"` (double quotes). Is that what you were thinking about ?